src/Parse.js

```diff
@@ -52,8 +52,6 @@ class Moralis extends MoralisWeb3 {
     } else {
       // Node environment
       if (!moralisSecret) {
```

Comment:
Reverse this check please as it is incorrect now

```suggestion
      if (moralisSecret) {
```